source/Halibut/Transport/TcpClientManager.cs

```diff
@@ -35,7 +35,7 @@ public void Disconnect(string thumbprint)
                 {
                     foreach (var client in tcpClients)
                     {
-                        client.Close();
+                        client.CloseImmediately();
```

Comment:
We could do with some tests that actually ensure this is working as expected when used on the HalibutRuntime. Tests on the extension method are great for all the variation testing but we want to know it really works for Halibut in some way as well.

Reply:
Would that be to try and test close is an immediate close, or is that to test that the connection is observed as closed?